README.md

```diff
@@ -39,23 +42,28 @@ void main() {
 
 > The `callbackDispatcher` needs to be either a static function or a top level function to be accessible as a Flutter entry point.
 
+The workmanager runs on a separate isolate from the main flutter isolate. Ensure to initialize all dependencies inside the `Workmanager().executeTask`. 
+##### Debugging tips
```

Comment:
please add newline above and below the headline

Reply:
Done